plugin.video.espn_3/resources/language/English/strings.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<strings>
+    <string id="30000">Play in XBMC. (Disable to use Browser) </string>
+    <string id="30001">Quality of Recorded Stream</string>
+    <string id="30003">Quality of Live Stream</string>
```

Comment:
In the next update, please changeover to strings.po from strings.xml, which is now deprecated.